python/pyspark/ml/feature.py

```diff
@@ -2490,7 +2490,8 @@ def setParams(self, inputCols=None, outputCol=None):
 
 
 @inherit_doc
-class VectorIndexer(JavaEstimator, HasInputCol, HasOutputCol, JavaMLReadable, JavaMLWritable):
+class VectorIndexer(JavaEstimator, HasInputCol, HasOutputCol, HasHandleInvalid, JavaMLReadable,
+                    JavaMLWritable):
     """
     Class for indexing categorical feature columns in a dataset of `Vector`.
 
```

Comment:
There is a TODO in the doc of `VectorIndexer`: `Add option for allowing unknown categories.`. I think we can remove it?